specs/data-grid/data-grid.spec.md

```diff
@@ -0,0 +1,301 @@
+# Data Grid
+
+## Overview
+
+The `data grid` component enables authors to display an array of data in a tabular layout.  The layout can include a "header" region where a title can be displayed for each column.  
+
+![](./images/grid.png)
+
+`Data grid` actually consists of a number of components that work together:
+- `<fast-data-grid>`:  The top level container element
+- `<fast-data-grid-header>`: Displays column titles
+- `<fast-data-grid-header-cell>`: Displays the title for a single column of cells
+- `<fast-data-grid-row>`: Displays a single row of data associated with a single record
+- `<fast-data-grid-cell>`: Displays a single cell of data within a row
+
+### Use Cases
+Any time an author wants to display tabular data.
```

Comment:
A few real world use cases wouldn't go amiss, some examples https://www.joelonsoftware.com/2000/05/09/the-process-of-designing-a-product/ you can see a few examples if you scroll down to "Imaginary users".